### PR TITLE
adds preloaded railguns

### PR DESCRIPTION
**Pull request by SonixApache**
added two preloaded railgun subtypes; a normal one charged to gib someone and a godslayer one with a t4 capacitor and a nullrod in it

### CODE REVIEW

**Comment by brndd** [comment]
Can we put these on the black market?

**Comment by boy2mantwicethefam** [comment]
This tbh

**Comment by SonixApache** [comment]
you're welcome to make the PR, i just tossed these in for bus

**Comment by RealestEstate** [comment]
>godslayer railgun

this kills the narsie

**Comment by Durgen** [comment]
making them admin spawnable kind of takes the magic out of them

Insane railguns are extremely rare, but some ADHD riddled admin is going to press buttons and change that

**Comment by SonixApache** [comment]
Day or two ago cult won, and I wanted to send in Doomguy with a fully loaded NT12 and a godslayer railgun to go and try to take on Nar-nar on his own
So I spawned;
 - a railgun
 - a nullrod
 - a loaded chief engineer toolbelt
 - a T4 capacitor that I had to varedit to have max charge
 - a rail assembly

And the guy didn't even know how to assemble it so he tossed it.
This'd be a quicker way to do that, that's why I made this. And I'm sure the ADD admins won't bus it in *that* often anyway, it's kinda pointless.
That aside, the normal preloaded railgun could actually be something worthwhile for TC trades / the meme market.

File: code/modules/projectiles/guns/projectile/constructable/railgun.dm [hunk 1]
```diff
@@ -291,7 +291,25 @@
...
 
+/obj/item/weapon/gun/projectile/railgun/preloaded
+	var/ammotype = /obj/item/weapon/coin/iron
+	var/capacitortype = /obj/item/weapon/stock_parts/capacitor/adv/super
+	
+/obj/item/weapon/gun/projectile/railgun/preloaded/New()
+	..()
+	loadedassembly = new /obj/item/weapon/rail_assembly
+	rails_secure = 1
+	loadedammo = new ammotype
```
Comment (ShiftyRail) [inline code/modules/projectiles/guns/projectile/constructable/railgun.dm]: Why isn't this `new ammotype(src)` ? Right now, it looks like this ammo is stored in nullspace.

Comment (SonixApache) [reply code/modules/projectiles/guns/projectile/constructable/railgun.dm]: lemme check that

**Pull request merged**